src/adapters/scene_category.js

```diff
@@ -27,4 +30,13 @@ export default class SceneCategory {
       )}
     )
   }
+
+  removeCategoryMarkers() {
+    this.markers.map(mark => mark.remove())
+    this.setOsmPoisVisibility('visible')
```

Comment:
For the visiblity, there is only two possible values (`visible` and `none`) ? If this is the case, there is better to use a boolean for param. With a method like this : `displayOsmPois(displayed)` where `displayed` is a boolean. And this is handled in the method with `const visiblity = displayed ? 'visible' : 'none'`.

If someday the string `visible` change, we only have to change this inside the `displayOsmPois` method 😌

Reply:
yup, ternary operator is coming